### PR TITLE
Add packages count at homepage

### PR DESCRIPTION
**Pull request by pkking**
Fix: #2770 
I just modify our theme and upstream theme, other theme (like redhat copr design) need some modifies as well :)

### CODE REVIEW

**Comment by pkking** [comment]
I made a modification to `pylintrc`, if its prohibit, please let me know :)

File: frontend/coprs_frontend/config/copr.conf [hunk 1]
```diff
@@ -171,7 +171,8 @@ HIDE_IMPORT_LOG_AFTER_DAYS = 14
 # OIDC_TOKEN_URL=""
 # OIDC_USERINFO_URL=""
 
-
+# Whether to show a total packages count at homepage
+# PACKAGES_COUNT = False
```
Comment (praiskup) [inline frontend/coprs_frontend/config/copr.conf:175]: We should have this set by default (FALSE) in `config.py`

Comment (pkking) [reply frontend/coprs_frontend/config/copr.conf:175]: > We should have this set by default (FALSE) in `config.py`
Done

**Comment by pkking** [comment]
Hi @praiskup, build failed on rawhide, could you help to diagnose to problem?

**Comment by FrostyX** [comment]
The rawhide failure is caused by #2776. It is unrelated to this PR, please ignore :-)

**Review by praiskup — APPROVED** [review]

**Comment by pkking** [comment]
> The rawhide failure is caused by #2776. It is unrelated to this PR, please ignore :-)

Tks!

**Pull request merged**